tools/util_generate_hex_from_json.cpp

```diff
@@ -0,0 +1,148 @@
+//
+// Created by Eric Passmore on 1/14/23.
+// Purpose: command line option to generate hex code from JSON
+//   may be used to generate serialization tests cases in other languages and external packages
+//
+
+#include "../src/abieos.h"
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <stdexcept>
+#include <unistd.h>
+#include <vector>
+
+// Main work done here four steps
+//      1) create empty context
+//      2) set the context
+//      3) parse json to binary
+//      4) create hex from bin
+// abiContractName: enum with contract names
+// schema: the name of the data type or reference to schema in the ABI contract
+// json: the name and values
+// verbose: flag to print out step by step messages
+std::string generate_hex_from_json(const char *abi_definition, const char *contract_name, const char *schema, const char *json, bool verbose) {
+    if (verbose) std::cerr << "Schema is: " << schema << " and json is " << json << std::endl << std::endl;
+
+    // create empty context
+    abieos_context_s *context = abieos_create();
```

Comment:
Since this is a simple execute and exit util not really needed, but would be nice to have `abieos_destroy` called in case someone uses this as an example on how to use abieos.

I would recommend something like:
```
  using unique_abieos = std::unique_ptr<abieos_context, decltype( &abieos_destroy )>;
  unique_abieos context( abieos_create(), &abieos_destroy );
```

Reply:
For future reference please put the `*` with the type: `abieos_context_s* context`